ledger/acctonline.go

```diff
@@ -304,18 +301,20 @@ func (ao *onlineAccounts) produceCommittingTask(committedRound basics.Round, dbR
 	if offset < dcr.offset {
 		dcr.offset = offset
 	}
+	dcr.oldBase = dbRound
 	return dcr
 }
 
 func (ao *onlineAccounts) consecutiveVersion(offset uint64) uint64 {
+	startIndex := len(ao.onlineRoundParamsData) - len(ao.deltas) - 1
 	// check if this update chunk spans across multiple consensus versions. If so, break it so that each update would tackle only a single
 	// consensus version.
-	if ao.versions[1] != ao.versions[offset] {
```

Comment:
remove the version and rountTotals from the `onlineAccounts` completely

Reply:
Where do you still see version / roundTotals? I already removed all of them.

Reply:
oh, I overlooked, thanks